### PR TITLE
Use await for creating claims

### PR DESCRIPTION
**Pull request by krobi64**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

## Description of Changes

Fixes poetapp/frost-api#243 
Needs poetapp/poet-js#96

Modify node to treat `createClaim` as async

### CODE REVIEW

**Comment by krobi64** [comment]
@lautarodragan This will fail until we release poet-js right?

**Comment by lautarodragan** [comment]
Yep. Using `await` is a harmless change since anything can be awaited, but the change in the interface is breaking.

**Comment by kennylavender** [comment]
@krobi64 can we get rid of the parse claim function inside here with this pr? It was added specifically because of the issue with the created/dateCreated being a Date object

https://github.com/poetapp/node/blob/master/src/Storage/ClaimController.ts#L205-L235

**Comment by lautarodragan** [comment]
re package-lock conflict: a simple way to solve it is `git checkout master -- package-lock.json` and then `rm -r node_modules` and `npm i`.

**Comment by lautarodragan** [comment]
Tagging `don't merge` for now since I think we're waiting for a reply to the review. @krobi64 lmk otherwise.

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,12 @@ services:
   - mongodb
   - rabbitmq
   - docker
+before_install:
+  - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
```
Comment (wzalazar) [inline .travis.yml]: I have a doubt here if we are adding this package, will it be necessary to add the same in the docker image?
@WesleyCharlesBlake ping!

Comment (lautarodragan) [reply .travis.yml]: good call @wzalazar — wondering the same now!

**Comment by lautarodragan** [comment]
Alright, I tried upping the fee and now I'm getting `64: too-long-mempool-chain. Code:-26`.

I left it retrying for a long while and it never succeeded in broadcasting a single transaction.

I'm going to try and get the BlockchainWriter migrated to Bitcoin Core so we can test this PR in regtest.

File: .travis.yml (hunk 1 above)
Comment (WesleyCharlesBlake) [reply .travis.yml]: @wzalazar @lautarodragan [https://github.com/poetapp/node/blob/master/Dockerfile#L3] the only packages being installed from apt in the Docker image is `rsync`.

Do we need to include the PPA `ppa:ubuntu-toolchain-r/test` ? what are we installing from this PPA? is it only `gcc-5 g++-5` ?

If this is a dep, we can build it into the Docker image by adding directives to the Dockerfile

Comment (wzalazar) [reply .travis.yml]: @WesleyCharlesBlake 
> Do we need to include the PPA ppa:ubuntu-toolchain-r/test ? what are we installing from this PPA? is it only gcc-5 g++-5 ?

yes, is a dependency for jsondl

for more context, I left this issue
https://github.com/digitalbazaar/jsonld.js/issues/254

**Pull request merged**